spec/requests/v1/users_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe '/v1' do
+  describe 'GET /v1/users/:user_id' do
+    it 'returns the details of a given user' do
+      user = FactoryBot.create(:user)
+
+      get "/v1/users/#{user.id}"
+
+      expect(response).to be_successful
+
+      expect(json['data']).to have_id(user.id)
+      expect(json['data'])
+        .to have_attribute(:name)
+        .with_value('User Name')
+      expect(json['data'])
+        .to have_attribute(:multiple_suppliers)
+        .with_value(false)
+    end
+
+    it 'returns the details of given user with more than one supplier' do
+      user = FactoryBot.create(:user)
+
+      user.suppliers << FactoryBot.create(:supplier)
```

Comment:
This is fine, but `user.suppliers << FactoryBot.create_list(:supplier, 2)` would also work.

Reply:
👍